cmd/runtimecfg/node-ip.go

```diff
@@ -250,10 +252,9 @@ func getSuitableIPs(retry bool, vips []net.IP, preferIPv6 bool, networkType stri
 		if !retry {
 			return nil, false, fmt.Errorf("Failed to find node IP")
 		}
-
-		log.Errorf("Failed to find a suitable node IP")
 		time.Sleep(time.Second)
```

Comment:
We need to make some modifications here. Always sleeping for a second is what is spamming the logs.